R/extract_table.R

```diff
@@ -36,8 +37,8 @@ extract_table <- function(.data,
   # Check tbl is valid ----
   assert_character(tbl)
 
-  if (length(tbl) > 1) {
-    stop("Only one table may be supplied.")
+  if (length(all_of(tbl)) > 1) {
```

Comment:
I think this is trivial enough that I'd be OK without a specific test

Reply:
From the `testthat` suite or remove an error message from the function entirely?

Reply:
Ok, associated `testthat` test removed 👍